colorizer_data/tests/test_converter.py

```diff
@@ -69,8 +75,8 @@ def test_handles_simple_csv(tmp_path):
             "description": "",
             "unit": "",
             "min": 0,
-            "max": 3,
-            "data": "feature_1.json",
+            "max": 2,
```

Comment:
Just curious, what's the reason for changing values in assertions?

Reply:
I marked the row that previously had the max value (3) as an outlier, which means its value won't be included in min/max calculation!